### PR TITLE
(DOCSP-26941) IA Refresh - Reference

### PR DESCRIPTION
**Pull request by nlarew**
## Pull Request Info

- Move reference content under `/reference` and update toctree
- Note: the following pages under reference will be moved or modified in future PRs
  - Known Issues & Workarounds
  - Service Limitations
  - Push Notifications [Deprecated]
  - Third-Party Services [Deprecated]
  - Find Your Project or App ID
  - Authenticate HTTP Client Requests
  - Upgrade a Shared Tier Cluster
  - Partition-Based Sync Mode

### Jira

- (DOCSP-26941) [Section] Reference

### Staged Changes

- [Staged](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-reference/)

### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-realm update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/reference/logs.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+   :titlesonly:
+
+   Authentication Logs </reference/logs/authentication>
+   Change Stream Logs </reference/logs/changestream>
+   Endpoint Logs </reference/logs/endpoint>
+   Function Logs </reference/logs/function>
+   Service Logs </reference/logs/service>
+   Trigger Logs </reference/logs/trigger>
+   Schema Logs </reference/logs/schema>
+   Sync Logs </reference/logs/sync>
```
Comment (mongodben) [inline source/reference/logs.txt:15]: shouuld we put links to all the pages under this? noticed you did for the other landing pages?

Comment (nlarew) [reply source/reference/logs.txt:15]: Oh yeah good catch I meant to do this. I was testing to see if the toctree still outputs a table on its own but, alas, no.

**Comment by mongodben** [comment]
why not kill [known issues and workarounds](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-reference/reference/issues-and-workarounds/) in this PR like we'd discussed?

**Comment by mongodben** [comment]
some non-blocking suggestions: 

1. in the spirit of decluttering the reference section and making it only reference info, i propose the following additional changes:

- move page [Authenticate HTTP Client Requests](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-reference/reference/authenticate-http-client-requests/) to Authenticate & Manage Users section
- Move page [Upgrade a Shared Tier Cluster](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-reference/reference/upgrade-shared-cluster/) to MongoDB Data Sources section

2. i think 'App Logs' section should remain within the 'Logs' section. while it's clearly reference information, it is very much scoped to the content of the Logs section, whereas much of the other stuff in the Reference section is more cross-cutting.

File: source/reference/logs.txt (hunk 1 above)
Comment (mongodben) [reply source/reference/logs.txt:15]: merp

**Review by mongodben — APPROVED** [review]
lgtm, tho 1 thought abt where the logs pages go that i'm curious about your opinion on

File: source/sync/get-started.txt [hunk 2]
```diff
@@ -225,3 +225,19 @@ with the backend app and can watch for and react to changes.
...
+Sync Fails To Enable Shortly After Cluster Creation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enabling Atlas Device Sync on a newly-created cluster may fail with an error
+message about parsing the URI. This happens because an SRV record hasn't
+propagated yet. While we work to provide more helpful diagnostics around
+this issue, there are two possible workarounds:
+
+- Wait five minutes, then enable Atlas Device Sync.
+- Unlink and :ref:`re-link the cluster <link-a-data-source>`.
```
Comment (mongodben) [inline source/sync/get-started.txt:243]: this feels like it should be in the 'Handle Errors' section. don't see a perfect location for this content, but perhaps add as an H2 section "Sync Fails To Enable Shortly After Cluster Creation" on https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-reference/sync/error-handling/errors/?

File: source/triggers/database-triggers.txt [hunk 3]
```diff
@@ -681,3 +681,32 @@ limit the number of fields that the Trigger processes by using a
...
+
+You can :manual:`restart </changeStreams/#resume-tokens>` the Trigger, but
+the Trigger will not process any events between the last processed event
+and the time that you restart the Trigger.
+
+.. seealso::
+
+   - For more information, refer to :ref:`Suspended Triggers <suspended_triggers>`.
+   - If you frequently see this error, consider
+     :ref:`Disabling event ordering <database-triggers-disable-event-ordering>`.
```
Comment (mongodben) [inline source/triggers/database-triggers.txt:712]: now that i see this content in the context of this page, i think that we don't need this as a dedicated section at all. it's very similar to the content in the [Suspended Triggers section](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-reference/triggers/database-triggers/#suspended-triggers). 

haven't done a super close read, but i think that there isn't any info in here that's not more thoroughly covered in the Suspended Triggers section.

**Review by mongodben — COMMENTED** [review]
back to you w a couple comments abt the known issues and workarounds. 

also some additional notes in our slack convo.

File: source/triggers/database-triggers.txt (hunk 3 above)
Comment (nlarew) [reply source/triggers/database-triggers.txt:712]: Yeah fair point. I'm all for deleting this entirely!

File: source/sync/get-started.txt (hunk 2 above)
Comment (nlarew) [reply source/sync/get-started.txt:243]: Yeah that's easy enough. I'm not sure how prevalent this is in the first place.

**Pull request merged**